nemo_text_processing/text_normalization/taggers/electronic.py

```diff
@@ -34,7 +34,12 @@ class ElectronicFst(GraphFst):
     def __init__(self):
         super().__init__(name="electronic", kind="classify")
 
-        username = pynutil.insert("username: \"") + NEMO_SIGMA + pynutil.insert("\"") + pynini.cross('@', ' ')
+        username = (
+            pynutil.insert("username: \"")
+            + pynini.closure(NEMO_ALPHA | NEMO_DIGIT | pynini.accep('.'))
```

Comment:
this would also accept ".abc@gmai.com" could you make sure it doesnt start with .?

Reply:
updated. @yzhang123, note that `test .b@gmail.com~test . b at g mail dot com` is still tagged unless we want to treat punct+email as a special case in the tokenizer_and_classify.py().